### PR TITLE
updated carousel, receiving dynamic data, and hover effect on it and item list

### LINKED ISSUE #61: Implement the /record/hot endpoint
Implement the /record/hot endpoint. 

https://app.swaggerhub.com/apis/glaubernespoli/DCI-SellYourRecords/1.0.0#/Records/findHotRecords

### LINKED ISSUE #72: Frontend :: Update the Carousel and Display Dynamic data in it

### LINKED ISSUE #81: Create the mouseOver effect on the album listed in the landing page, to improve and make more friendly the user interface

### LINKED ISSUE #85: Carousel

### PR DESCRIPTION
**Pull request by santiagoejm**

### CODE REVIEW

**Review by arifur-ra — DISMISSED** [review]
Done, Everything looks fine

File: frontend/src/containers/Home/carousel/Carousele.jsx [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+      <Grid container>
+        <Grid item xs={12}>
+          <Carousel
+            animation="slide"
+            interval={3000}
+            navButtonsAlwaysVisible
+            className={classes.root}
+          >
+            {data.map((item) => (
+              // eslint-disable-next-line no-underscore-dangle
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Carousele.jsx]: this line is not needed with the rule for _id disabled

File: frontend/src/containers/Home/carousel/Item.jsx [hunk 2]
```diff
@@ -1,44 +1,80 @@
...
-          <Typography align="center" gutterBottom paragraph variant="caption">
-            {item.description}
-          </Typography>
-        </Grid>
+  const context = useContext(MyContext);
+  const { setItem } = context;
+  const navigate = useNavigate();
+
+  const clickHandle = () => {
+    // eslint-disable-next-line no-underscore-dangle
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Item.jsx]: same here

File: frontend/src/containers/Home/carousel/Carousele.jsx [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
-          <Item key={item.id} item={item} />
-        ))}
-      </Carousel>
-    </Grid>
-  </Grid>
-);
+const Carousele = () => {
+  const classes = useStyles();
+
+  const { data, error, isLoading } = useAxios('get', '/record');
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Carousele.jsx]: still using the wrong endpoint to capture the records, should be /record/hot

File: frontend/src/containers/Home/carousel/Carousele.jsx [hunk 4]
```diff
@@ -1,17 +1,45 @@
...
+      <div>
+        <p>{error.message}</p>
+      </div>
+    );
+  }
+
+  return (
+    <Paper>
+      <Grid container>
+        <Grid item xs={12}>
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Carousele.jsx]: as shown, gris and paper not needed

File: frontend/src/containers/Home/carousel/Item.jsx [hunk 5]
```diff
@@ -1,44 +1,80 @@
...
-          </Typography>
-        </Grid>
 
-        <Grid item>
-          <Typography align="center" gutterBottom paragraph variant="caption">
-            {item.description}
-          </Typography>
-        </Grid>
+  const context = useContext(MyContext);
+  const { setItem } = context;
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Item.jsx]: as we spoke, context is being set here and not used. delete

File: frontend/src/containers/Home/carousel/Item.jsx [hunk 6]
```diff
@@ -1,44 +1,80 @@
...
+  const navigate = useNavigate();
+
+  const clickHandle = () => {
+    // eslint-disable-next-line no-underscore-dangle
+    navigate(RecordRoute(item._id));
+    setItem(item);
```
Comment (glaubernespoli) [inline frontend/src/containers/Home/carousel/Item.jsx]: delete

**Review by glaubernespoli — CHANGES_REQUESTED** [review]

**Review by glaubernespoli — APPROVED** [review]
(Y)

**Review by glaubernespoli — COMMENTED** [review]
fhjfghj

**Comment by Manonmani-s** [comment]
> fhjfghj

Glauber, happy to see the review.

**Pull request merged**